lib/bulk-apply-applications.js

```diff
@@ -170,17 +181,18 @@ function bulkApplyApplications (args, callback) {
     if (updateAction === 'disapprove' || updateAction === 'checkin' || !available) return done(null, applications);
```

Comment:
A check to !available here means that a user won't be notified of going back to pending (for example) if the queue is already full.
This is faulty, not your fault, but needs to get done.